server/game/GameActions/index.js

```diff
@@ -62,7 +63,8 @@ const GameActions = {
         return new SimultaneousAction(actions);
     },
     standCard: props => new AbilityAdapter(StandCard, props),
-    takeControl: props => new AbilityAdapter(TakeControl, props)
+    takeControl: props => new AbilityAdapter(TakeControl, props),
+    returnCardToDeck: props => new AbilityAdapter(ReturnCardToDeck, props),
```

Comment:
Try to keep this (and the corresponding `require` at the top) in alphabetical order.

Reply:
fixed